.pre-commit-config.yaml

```diff
@@ -32,5 +32,4 @@ repos:
     hooks:
       - id: pylint
         args: [ "./**/*.py" ]
-        additional_dependencies:
-          - pyenchant
```

Comment:
Could you explain this change to me?

Reply:
oh sorry! my bad, I may have changed it unintentionally. everything should be back to normal now.